pkg/koordlet/resourceexecutor/reader.go

```diff
@@ -38,7 +38,8 @@ type CgroupReader interface {
 	ReadMemoryNumaStat(parentDir string) ([]sysutil.NumaMemoryPages, error)
 	ReadCPUTasks(parentDir string) ([]int32, error)
 	ReadPSI(parentDir string) (*PSIByResource, error)
-	ReadMemoryColdPageUsage(parentDir string) (uint64, error)
+	ReadMemoryColdPageUsage(parentDir string, boundary int) (uint64, error)
```

Comment:
Currently, there is no case to use an individual boundary for each cgroup. So the boundary should be a component-level/node-level config, instead of an argument of the cgroup reader method which may confuse the caller.